inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/forms/FormQuestion.java

```diff
@@ -0,0 +1,101 @@
+package uk.ac.ucl.rits.inform.informdb.forms;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import uk.ac.ucl.rits.inform.informdb.TemporalCore;
+import uk.ac.ucl.rits.inform.informdb.annotation.AuditTable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.Instant;
+
+/**
+ * Represents the Question in a form (basically the metadata for a form answer).
+ * It may not literally be a question, eg. "Limb", you can think of that as a prompt.
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@AuditTable
+public class FormQuestion extends TemporalCore<FormQuestion, FormQuestionAudit> {
+    /**
+     * \brief Unique identifier in EMAP.
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long formQuestionId;
+
+    /**
+     * \brief String identifier from the source system of the question.
+     * Eg. Epic it would be the Smart Data Element ID (HLV 40).
+     */
+    @Column(nullable = false, unique = true)
+    private String formQuestionSourceId;
+
+    /**
+     * \brief Smart Data Element name (ie. the "Question")
+     * HLX .2
+     */
+    private String formQuestionConceptName;
```

Comment:
there's a lot of prefixing with the table name, worth removing the prefix as it doesn't give us more information - and the same for other fields in this and other classes
```suggestion
    private String conceptName;
```